build.cake

```diff
@@ -75,7 +75,7 @@ Task("Patch-Assembly-Info")
         Version = version,
         FileVersion = version,
         InformationalVersion = semVersion,
-        Copyright = "Copyright (c) Patrik Svensson, Mattias Karlsson and contributors"
+        Copyright = "Copyright (c) Patrik Svensson, Mattias Karlsson, Gary Ewan Park and contributors"
```

Comment:
@devlead related to this, should these also be updated?

![image](https://cloud.githubusercontent.com/assets/1271146/10970050/05b3d1a0-83c4-11e5-8319-2c7cf57c04e6.png)